uitools/examples/cpp_quick/src/main.cpp

```diff
@@ -61,21 +60,19 @@ int main(int argc, char* argv[])
     Esri::ArcGISRuntime::ArcGISRuntimeEnvironment::setApiKey(apiKey);
   }
 
-  QQuickStyle::addStylePath("qrc:///esri.com/imports/");
```

Comment:
does calcite styling still work with this removed?

Reply:
I think it is still fine - here's some details on why it needed removed: https://doc.qt.io/qt-6/qtquickcontrols-changes-qt6.html#custom-styles-are-now-proper-qml-modules